pallets/rc-migrator/src/lib.rs

```diff
@@ -1137,6 +1160,59 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		/// Returns `true` if the migration is ongoing and the Asset Hub has not confirmed
+		/// processing the same number of XCM messages as we have sent to it.
+		fn has_excess_unconfirmed_xcm(current: &MigrationStageOf<T>) -> bool {
+			if !current.is_ongoing() {
+				return false;
+			}
+			let (sent, processed) = XcmDataMessageCounts::<T>::get();
+			const ALLOWED_UNCONFIRMED: u32 = 0;
```

Comment:
0 here means that we do it synchronously?  
Can you maybe pull this constant out and add a doc comment please?